doc/rados/operations/health-checks.rst

```diff
@@ -851,7 +853,8 @@ To change this, run the following command:
    ceph config set global bdev_stalled_read_warn_lifetime 10
    ceph config set global bdev_stalled_read_warn_threshold 5
 
-this may be done surgically for individual OSDs or a given mask
+this may be done surgically for individual OSDs or a given mask, e.g.
```

Comment:
s/e.g./for example/

I personally like "e.g.", but I think that there are a lot of people who don't know the difference between "e.g." and "i.e." and I don't want to confuse them.